src/test/kotlin/uk/gov/dluhc/notificationsapi/rest/GenerateBespokeCommTemplatePreviewIntegrationTest.kt

```diff
@@ -264,6 +264,7 @@ internal class GenerateBespokeCommTemplatePreviewIntegrationTest : IntegrationTe
         val languageDto = languageMapper.fromApiToDto(language)
         val expectedWhatYouNeedToDo = requestBody.personalisation.whatToDo != null
         val expectedDeadline = deadlineMapper.toDeadlineString(requestBody.personalisation.deadlineDate!!, requestBody.personalisation.deadlineTime!!, languageDto, sourceTypeMapper.toFullSourceTypeString(sourceType, languageDto))
+        val expectedAn = expectedPersonalisationFullSourceType == "overseas vote"
```

Comment:
I think for the test it would be best to hardcode a value here instead of replicating the mapping logic